lang/en.json

```diff
@@ -1,17 +1,18 @@
 {
 
-  "dfreds-convenient-effects.conditionAlmostdead": "Almost dead",
-  "dfreds-convenient-effects.conditionBlinded": "Blind",
+  "dfreds-convenient-effects.conditionAlmostDead": "Almost dead",
+  "dfreds-convenient-effects.conditionBlinded": "Blinded",
   "dfreds-convenient-effects.conditionCharmed": "Charmed",
-  "dfreds-convenient-effects.conditionConcentrating": "Concentrating",
+  "dfreds-convenient-effects.conditionConcentration": "Concentration",
   "dfreds-convenient-effects.conditionDead": "Dead",
   "dfreds-convenient-effects.conditionDeafened": "Deafened",
   "dfreds-convenient-effects.conditionDiseased": "Diseased",
-  "dfreds-convenient-effects.conditionExhausted1": "Exhaustion 1",
-  "dfreds-convenient-effects.conditionExhausted2": "Exhaustion 2",
-  "dfreds-convenient-effects.conditionExhausted3": "Exhaustion 3",
-  "dfreds-convenient-effects.conditionExhausted4": "Exhaustion 4",
-  "dfreds-convenient-effects.conditionExhausted5": "Exhaustion 5",
+  "dfreds-convenient-effects.conditionExhaustion": "Exhaustion",
```

Comment:
I think this is unnecessary as all the exhaustion levels are defined

Reply:
Yes, I imagined it